crates/cargo-test-support/src/compare.rs

```diff
@@ -222,6 +222,12 @@ fn substitute_macros(input: &str) -> String {
         ("[REPLACING]", "   Replacing"),
         ("[UNPACKING]", "   Unpacking"),
         ("[SUMMARY]", "     Summary"),
+        ("[DESTINATION]", " Destination"),
+        ("[INSTALL]", "     Install"),
+        ("[UPGRADE]", "     Upgrade"),
+        ("[DOWNGRADE]", "   Downgrade"),
+        ("[UPTODATE]", "  Up-to-date"),
+        ("[REINSTALL]", "  Re-install"),
```

Comment:
imo the dry-run should generally match the regular behavior.

If we want to improve the `cargo install` output, that is a change on its own to be evaluated.